src/test/java/org/walkmod/findbugs/transformations/AddFinalKeywordVisitorTest.java

```diff
@@ -0,0 +1,33 @@
+package org.walkmod.findbugs.transformations;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import org.junit.Assert;
+import org.junit.Test;
+import org.walkmod.javalang.ASTManager;
+import org.walkmod.javalang.ast.CompilationUnit;
+
+import java.io.File;
+import java.io.IOException;
+
+public class AddFinalKeywordVisitorTest {
+    @Test
+    public void shouldAddFinalKeywordToParameters() throws Exception {
+        final String base = "final-parameters/without-final";
+        final String input = base + "/input";
+        final String expected = base + "/expected";
+        final CompilationUnit cu = ASTManager.parse(getResourceFile(input));
+        final AddFinalKeywordVisitor visitor = new AddFinalKeywordVisitor();
+        visitor.visit(cu, null);
+        Assert.assertEquals(getResourceString(expected), cu.toString());
```

Comment:
The only comment - let's use assertj everywhere to be consistent :)

Reply:
Done :) Never used assertj